app/web/src/components/Navigation/Navigation.tsx

```diff
@@ -127,14 +166,68 @@ const useStyles = makeStyles((theme) => ({
     alignSelf: "center",
     fontWeight: "bold",
   },
+  menu: {
+    boxShadow: theme.shadows[1],
+    minWidth: "12rem",
+  },
+  menuPopover: {
+    transform: "translateY(2rem)",
+  },
+  menuTitle: {
+    fontWeight: "bold",
+    position: "relative",
+    pointerEvents: "none",
+    "&:after": {
+      content: "''",
+      position: "absolute",
+      left: 16,
+      bottom: 0,
+      borderBottom: "1px solid black",
+      width: 32,
+    },
+  },
+  menuItem: {
+    width: "100%",
+  },
+  menuItemMessage: {
+    display: "flex",
+    flexDirection: "row",
+    justifyContent: "space-between",
+    alignItems: "center",
+  },
+  notificationCount: {
+    color: grey[500],
+    fontWeight: "bold",
+  },
+  menuBtn: {
+    display: "flex",
+    flexDirection: "row",
+    alignItems: "center",
+    border: `1px solid ${grey[300]}`,
+    borderRadius: "35%",
+    backgroundColor: grey[200],
+    padding: 4,
+    cursor: "pointer",
+  },
+  avatar: {
+    height: 30,
+    width: 30,
+  },
+  badge: {
+    right: "-4px",
+    top: "4px",
+  },
 }));
 
 export default function Navigation() {
   const authClasses = useAuthStyles();
   const classes = useStyles();
   const authenticated = useAuthContext().authState.authenticated;
   const [open, setOpen] = React.useState(false);
+  const [menuOpen, setMenuOpen] = React.useState<boolean>(false);
```

Comment:
```suggestion
  const [menuOpen, setMenuOpen] = useState(false);
```
Importing useState directly is a style thing to match the rest of the repo, no need for the typing since it's inferred.

Reply:
I always do that but i read some code and i'm pretty sure i see React.useState somewhere, should i change that too ?